browser_history/cli.py

```diff
@@ -64,21 +64,21 @@ def main():
                     break
             browser_class = getattr(browsers, selected_browser)
         except AttributeError:
-            print(f'Browser {args.browser} is unavailable. Check --help for available browsers')
+            utils.logger.error(f'Browser {args.browser} is unavailable. Check --help for available browsers')
```

Comment:
You should use `logging`'s own formatting instead of f-strings ([reference](https://docs.python.org/3/howto/logging.html#logging-variable-data)).
(`pylint` gives a warning for this)
```suggestion
            utils.logger.error('Browser %s is unavailable. Check --help for available browsers',
                               args.browser)
```